### DISCUSSION TITLE
Stores Tab

### DISCUSSION THREAD

**Discussion by james0r** [alpine-collective/alpinejs-devtools#375] [Remnant]
I know there was another discussion thread for Spruce but just wanted to bump this since Alpine.store() is part of core since v3.

Is this on a/the roadmap?

Thanks!

**Comment by james0r** [comment]
> I know there was another discussion thread for Spruce but just wanted to bump this since Alpine.store() is part of core since v3.
> 
> Is this on a/the roadmap?
> 
> Thanks!

Also mentioned on Alpine core discussions. Anyone finding this may want to comment or emote there https://github.com/alpinejs/alpine/discussions/3505.

**Comment by HugoDF** [comment]
There's no roadmap as such.

I'm running a survey/GH issue to suss out what we can do on this project.

The "stores" tab is part of the "future features" section of the survey 👉  https://github.com/alpine-collective/alpinejs-devtools/issues/431
